int32 grip_action
---
bool pos_ok 
